shortlivedpool.go

```diff
@@ -0,0 +1,17 @@
+// +build shortlivedpool
+
+package bytebufferpool
+
+import (
+	"log"
+
+	"github.com/gallir/shortlivedpool"
```

Comment:
I don't recommend using shortlivedpool seeing as it [stack allocates for every operation](https://github.com/gallir/shortlivedpool/issues/2) and [has data race issues](https://github.com/gallir/shortlivedpool/issues/3).